source/patterns/@aws-solutions-constructs/core/test/s3-bucket.test.ts

```diff
@@ -154,3 +154,177 @@ test('test createAlbLoggingBucket()', () => {
     BucketName: 'test-name'
   });
 });
+
+test('Test bucket policy that only accepts SSL requests only', () => {
```

Comment:
Let's add 2 more tests - 
-SSL is enforced when bucketProps is not provided
-SSL is enforced when bucketProps is provided, but enforceSSL is not set

When we're talking a security issue we want to ensure that every possibility is explicitly tested.